packages/notifications/graphql/schema.js

```diff
@@ -5,9 +5,11 @@ schema {
 
 type mutations {
   upsertDevice(token: ID!, information: DeviceInformationInput!): Device!
-  rollDeviceToken(oldToken: String!, newToken: String!): Device!
 
-  # user's can remove their devices
+  # deprecated: not used in app anymore, evict if no API calls tracked anymore
```

Comment:
There is this beautiful directive in GraphQL schemas: `@deprecated`. I can see this used here instead of having deprecation notice in a comment.

```
rollDeviceToken(oldToken: String, newToken: String!): Device! @deprecated(reason: "Lorem ipsum dolor sit amet")
```

<img width="338" alt="Bildschirmfoto 2019-08-28 um 05 55 17" src="https://user-images.githubusercontent.com/331800/63824729-73494f00-c958-11e9-8b28-8ee6b6038ea3.png">

As far as I can tell it bears no impact on querying or mutating but is soley for [documentaton and tooling purposes](https://graphql.github.io/graphql-spec/June2018/#sec--deprecated). It even allows Markdown formatted string in `reason` argument 🌟

Reply:
Thanks for pointing it out. I tried to adopt this style when the definition of deprecated fields changed in graphql sometime last year. Sadly this directive is not available for enums nor mutation params, which let me to use the description. For the sake of beauty I now adopted it where possible.